doc/api/console.markdown

```diff
@@ -9,9 +9,9 @@ The module exports two specific components:
 
 * A `Console` class with methods such as `console.log()`, `console.error()` and
   `console.warn()` that can be used to write to any Node.js stream.
-* A global `console` instance configured to write to `stdout` and `stderr`.
-  Because this object is global, it can be used without calling
-  `require('console')`.
+* A global `console` instance configured to write to process `stdout` and
```

Comment:
Sorry, I should have clarified. I meant use either `stdout`/`stderr` or `process.stdout`/`process.stderr` everywhere. Adding 'process' here without it being inside the backticks makes it look odd.